src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
I want the middleware always be activated on every request.

If you specify the `path` like `/api/graphql` only if i run a request against that path i will activate the middleware. 
Otherwise without specify the `path` option i'm able to query against a specic `URI` or against a specific `HEADER` without any assumption on the URI

```
GET /api/graphql //activate middleware by URI
POST /api/graphql //activate middleware by URI
POST `/` --header 'content-type: application/graphql'
```

I think this is a matter of preference..